pennylane/qnode_new/decorator.py

```diff
@@ -16,20 +16,30 @@
 """
 from functools import lru_cache
 
+from .base import BaseQNode
 from .cv import CVQNode
+from .device_jacobian import DeviceJacobianQNode
 from .jacobian import JacobianQNode
-from .qnode import QNode as BaseQNode
 from .qubit import QubitQNode
 
 
-def QNode(
-    func, device, *, interface="autograd", mutable=True, differentiable=True, properties=None
-):
+PARAMETER_SHIFT_QNODES = {"qubit": QubitQNode, "cv": CVQNode}
+
+
+def QNode(func, device, *, interface="autograd", mutable=True, diff="best", properties=None):

```

Comment:
I believe that sticking to writing out each word in the names leads to better readability. Therefore having ``function``, ``differentiation_method``, etc. would be preferable. E.g. just by looking at the signature, ``diff="best"`` might not provide too much information. If an abbreviated name is picked, however, maybe ``diff_method`` is quite short and refers to what it is a bit better.

Reply:
I like `diff_method`, will make that change

Reply:
Totally on the same page as Antal here. Letters are free, so there is no issue in using them :) I think `diff_method` is good though.

Reply:
Updated to `diff_method`.